src/api/index.md

```diff
@@ -2165,6 +2165,16 @@ type: api
   })
   ```
 
+  Sort using two keys:
+
+  ``` html
+  <ul>
+    <li v-for="user in users | orderBy ['lastName', 'firstName']">
```

Comment:
Just so you know you can't use inline arrays like this as filter arguments. It breaks the parsing currently.

I forgot to mention that `orderBy` should support inline multiple sortKeys like

``` html
<li v-for="user in users | orderBy 'lastName' 'firstName'"></li>
```

One of the reason being that the above inline array wouldn't work.